doc/Type/List.pod6

```diff
@@ -245,7 +245,7 @@ these invocations in a result list.
 
 Unlike C<map> it flattens non-itemized lists and arrays, so
 
-    say ((1, 2), <a b>).flatmap(&uc).join('|');     # 1|2|A|B
+    say ((1, 2), <a b>).flatmap(&uc).join('|');     # 1 2|A B
 
```

Comment:
Then someone should fix this exact line, obviously. New issue, perhaps? I can't make up a good example right now.